tests/test.js

```diff
@@ -62,3 +75,57 @@ describe("Embroider build", function () {
     });
   }
 });
+
+describe("Build error on missing integrity hash for external resources", function () {
+  let buildOutput;
+
+  beforeEach(async function () {
+    this.timeout(60000);
+    try {
+      await execa("pnpm", ["build"], {
+        cwd: path.join(
+          __dirname,
+          "..",
+          "test-apps",
+          "external-resource-missing-integrity-hash",
+        ),
+      });
+    } catch (error) {
+      buildOutput = error.message;
+    }
+  });
+
+  it("Throws an error if integrity hash is missing", function () {
+    expect(buildOutput).to.include(
+      "🚨🚨 The following external resources do not have an integrity hash:\n" +
+        '\n<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>\n',
+    );
+  });
+});
+
+describe("No build error when integrity hash present for external resources", function () {
```

Comment:
I think we should extend that test by also cover that the `integrity` and `crossorigin` attributes are not modified.